packages/services/src/epcisEvents/types.ts

```diff
@@ -60,3 +60,13 @@ export interface ITransformationEvent extends IContext {
   epcisTransformationEvent: IEntityIssue;
   productTransformation: IProductTransformation;
 }
+
+export interface IAggregationEvent {
```

Comment:
I see in other events we also receive data similar to this interface, should we change to the common name and apply it for others?